cmake/cflags.cmake

```diff
@@ -3,7 +3,7 @@
 # not set CMAKE_CXX_FLAGS globally because gtest is not warning-clean.
 if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "\\w*Clang")
     if (ENABLE_CXX_WERROR)
-        set(CMAKE_CXX_FLAGS "-Werror ${CMAKE_CXX_FLAGS}")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
This can be removed in the final merge. I disabled this before making any changes just to get the main branch to compile.